content/en/apps/features/integrations/oppia.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "OppiaMobile"
+weight: 3
+description: >
+   Integrate CHT core with OppiaMobile’s learning management platform
+keywords: oppia
+relatedContent: >
+  apps/examples/learning-care
+  apps/guides/integrations/oppia
+---
+
+[OppiaMobile](https://digital-campus.org/oppiamobile) is an open source mobile learning platform built by Digital Campus specially designed for delivering learning content, multimedia, and quizzes in low-broadband settings. All the content and activities can be accessed and used when no internet connection is available, and users can earn points and badges for completing activities and watching videos. To learn more about the platform, check out the [overview](https://digital-campus.org/oppiamobile/developers), OppiaMobile on [Github](https://github.com/DigitalCampus), and their [documentation site](http://oppiamobile.readthedocs.io/en/latest). You can also join the OppiaMobile [Community Discussion Board](https://community.oppia-mobile.org).
+
+
+## Overview
+This documentation describes how the CHT and OppiaMobile can integrate to provide a learning and care experience for community health workers and other health care providers. It demonstrates how both apps link to one another to provide a seamless user experience, describes features of both applications, and the required configuration adjustments.
+
+We provide a detailed example of the CHT<>OppiaMobile integration, including how to access the learning material, an overview of the functionalities within educational modules, and post-course assessment and supervisor support.
+
+
+## Features
+This integration leverages the **remote onboarding, task & scheduling, and dashboard features** of the CHT core framework with the **curated, multimedia educational content** available via OppiaMobile’s learning platform. 
```

Comment:
I'm not really sure what the `dashboard features` are in the CHT core.

Reply:
supposed to be targets, good catch